src/size_label_prs.py

```diff
@@ -20,6 +20,9 @@ def size_label_prs(data, pull):
     elif additions < 500:
         label = "Size/XL"
 
+    else:
```

Comment:
WDYT about set a comment as well about this PR is big ?

Reply:
I like that idea, was thinking of how to put it in the label, but comment would be better :)